packages/canary-react/src/component-tests/IcDataTable/IcDataTable.cy.tsx

```diff
@@ -2708,7 +2719,7 @@ describe("IcDataTable with truncation", () => {
         const [truncationPattern, setTruncationPattern] =
           useState<IcDataTableTruncationTypes>(undefined);
         const [rowHeight, setRowHeight] = useState<number>();
-        const [updatedRows, setUpdatedRows] = useState<number>(5);
+        const [updatedRows] = useState<number>(5);
```

Comment:
minor one, but is there any need to useState if value is not going to be updated?